.github/workflows/deploy-demo.yml

```diff
@@ -65,3 +65,9 @@ jobs:
           --var ADMIN_CLIENT_SECRET="$ADMIN_CLIENT_SECRET"
           --var AWS_ACCESS_KEY_ID="$AWS_ACCESS_KEY_ID"
           --var AWS_SECRET_ACCESS_KEY="$AWS_SECRET_ACCESS_KEY"
+
+    - name: Deploy egress proxy
```

Comment:
Can this happen conditionally on changes to `deploy-config/egress_proxy`?

Reply:
And to check, since I'm not sure: does the order of deployment matter, such that this should come before the app deployment?

Reply:
This should come after the app deployment, since the app needs to exist before the proxy can be connected to it.